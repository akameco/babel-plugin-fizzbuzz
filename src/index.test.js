// @flow
import pluginTester from 'babel-plugin-tester'
import plugin from '.'

pluginTester({
  plugin,
  snapshot: true,
  tests: [
    `
  for (var i=0; i<100; i++) {
    console.log(i)
  }
  `,
    `
  for (var j=0; j<100; j++) {
    console.log(j)
  }
  `,
  ],
})
