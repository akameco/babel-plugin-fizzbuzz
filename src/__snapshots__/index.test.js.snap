// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-plugin-fizzbuzz 1. babel-plugin-fizzbuzz: 1. babel-plugin-fizzbuzz 1`] = `
"
for (var i=0; i<100; i++) {
  console.log(i)
}

      ↓ ↓ ↓ ↓ ↓ ↓

for (var i = 0; i < 100; i++) {
  console.log(i % 15 === 0 ? 'FizzBuzz' : i % 3 === 0 ? 'Fizz' : i % 5 === 0 ? 'Buzz' : i);
}
"
`;

exports[`babel-plugin-fizzbuzz 2. babel-plugin-fizzbuzz: 2. babel-plugin-fizzbuzz 1`] = `
"
for (var j=0; j<100; j++) {
  console.log(j)
}

      ↓ ↓ ↓ ↓ ↓ ↓

for (var j = 0; j < 100; j++) {
  console.log(j % 15 === 0 ? 'FizzBuzz' : j % 3 === 0 ? 'Fizz' : j % 5 === 0 ? 'Buzz' : j);
}
"
`;
